src/platform_impl/macos/window_delegate.rs

```diff
@@ -172,9 +172,18 @@ declare_class!(
             let filenames = pb.propertyListForType(unsafe { NSFilenamesPboardType });
             let filenames: Id<NSArray<NSString>, Shared> = unsafe { Id::cast(filenames) };
 
+            let dl: NSPoint = unsafe { msg_send![sender, draggingLocation] };
+            let rect = NSRect {
+                origin: dl,
+                ..Default::default()
+            };
+            let y = util::bottom_left_to_top_left(rect);
+            let scale_factor = self.window.scale_factor();
+            let position = LogicalPosition::<f64>::from((dl.x, y)).to_physical(scale_factor);
```

Comment:
Hmm, nope, turns out the `y` coordinate was still wrong. I've pushed a proper fix for this now.

Reply:
thanks